scripts/oc-cluster.sh

```diff
@@ -8,8 +8,8 @@ OPENSHIFT_CLIENT_BINARY_URL=${OPENSHIFT_CLIENT_BINARY_URL:-'https://github.com/o
 
 sudo service docker stop
 
-sudo sed -i -e 's/sock/sock --insecure-registry 172.30.0.0\/16/' /etc/default/docker
-sudo cat /etc/default/docker
+sudo sed -i -e 's/"mtu": 1460/"mtu": 1460, "insecure-registries": ["172.30.0.0\/16"]/' /etc/docker/daemon.json
```

Comment:
daemon.json +1

Reply:
the same change is already done in https://github.com/openshift/odo/pull/2913

Reply:
As both the pr needs that change and we don't have this on master, So I was just checking if it works. Once it goes in I will rebase the pr and we are good to go.